new temp blog post for our Zune giveaway (#2469)
alcionai/corso: website/blog/2023-2-10-zune.md

We've worked hard making the industry's only free and open-source tool for backing up Microsoft 365 data.
And that's why we're not letting anything from Microsoft die, certainly not 2006's best personal media player, the Zune.

…look, this all made sense when I bought the Zune and found someone to refurbish it. Just play along with me here, okay?
<!-- truncate -->

![A gif from the show 30 Rock with a character insisting 'beepers are about to make a big comeback, technology is cyclical'](./images/cyclical.gif)

## A technology whose time has come

In many ways, the Zune was ahead of its time. Its wi-fi could stream music, and share your own media with other
Zunes on the local network.

It also, to my knowledge, was the first personal media player to come in brown. This Zune isn't brown, it's black,
but they did make brown ones.

![image of a Microsoft Zune By BulbousSum - Own work, CC BY-SA 4.0, https://commons.wikimedia.org/w/index.php?curid=120341960](./images/brown_zune.jpeg)

*Look at that, brown. What a time to be alive.*

And not only can you load a Zune with MP3 or WMA files, but it's also got an FM radio.

![image of a zune tuning in FM radio](./images/radio_zune.jpeg)

> *fm radio, certainly nothing will replace that*

Reviewer: ```suggestion
*FM radio, certainly nothing will replace that!*
```

This is a good thing since you'll have to run Windows Vista or earlier to load any media files onto the actual player.

## All right Nica, I can't possibly get more excited, how can I win this prize?

> It will take you about 20 minutes. Just:

Reviewer: ```suggestion
It will take you about 15 minutes (or less!). Just:
```

> * [give Corso a try](https://corsobackup.io/docs/quickstart/)

Reviewer: ```suggestion
* [Give Corso a try](https://corsobackup.io/docs/quickstart/)
```

> * fill out the [Corso feedback form](https://forms.microsoft.com/r/mRVNKqeKDp)

Reviewer: ```suggestion
* Fill out the [Corso feedback form](https://forms.microsoft.com/r/mRVNKqeKDp)
```
